pkg/gatherers/clusterconfig/gather_silenced_alerts_test.go

```diff
@@ -0,0 +1,70 @@
+package clusterconfig
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+)
+
+type mockAlertsClient struct {
+	data []byte
+}
+
+func (c *mockAlertsClient) RestClient(t *testing.T) *rest.RESTClient {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if c.data == nil {
+			w.WriteHeader(http.StatusNotFound)
+		} else {
+			w.WriteHeader(http.StatusOK)
+			// nolint: errcheck
+			w.Write(c.data)
+		}
+	}))
+	return testRESTClient(t, ts)
```

Comment:
This is defined in `node_logs_test.go`. I think this is not very "friendly" approach. Imagine that you're working and changing the `node_logs_test.go` and suddenly you also have failures in other unrelated test.

Reply:
So, I agree. I'm considering to create a `testing/` folder at the root project (not sure if this is the best approach, but seems to be very common) to add some test utilities there. But, I'm not sure how many test utilities we have lost/shared across the project. I want to handle it in a separate task.

Reply:
Another approach is to have some repeated code, between the tests, leaving them as independent as possible. But, as I said. I'm not sure yet, I want to check how many dependencies, and testing utilities we have before taking any decision.

Reply:
I think little copying in this case is OK, second option of moving the helper functions and reuse them is also OK.